core/validation/specific_validations/towns_fund_round_four.py

```diff
@@ -124,17 +132,19 @@ def validate_funding_profiles_funding_source(
     if len(invalid_rows) > 0:
         return [
             TownsFundRoundFourValidationFailure(
-                tab="Funding Profiles",
+                sheet="Funding",
                 section=f"Project Funding Profiles - Project {get_project_number(row['Project ID'])}",
+                column="Funding Source Type",
                 message=f'For column "Funding Source", you have entered "{row["Funding Source Type"]}" which isn\'t '
                 f"correct. You must select an option from the list provided",
+                indexes=row.name,
```

Comment:
How come we use `name` for this?

Reply:
I think it's because row.index will not return an integer of the index but instead a series of index labels, but row.name returns whatever the value in the index is for that row. I think it should therefore also work on the index isn't 0-indexed, but perhaps we should test for when it is not as well in this case?